### DISCUSSION TITLE
Dig Octa Ethernet configuration

### DISCUSSION THREAD

**Discussion by marcobrianza** [forkineye/ESPixelStick#747] [Q&A] [answered]
I have some issues with a dig octa board and ethernet connection, before reporting the strange behaviour I want to clarify if the configuration I have is correct. flashing the latest CI firmware I get a couple of red warnings on some pin as you can see in the screenshot. What should be the correct value?

![ethernet](https://github.com/forkineye/ESPixelStick/assets/1008278/fb0feade-2abf-4a09-a71b-7bab79994e34)

**Comment by MartinMueller2003** [comment, marked as answer]
Both of those are valid. I need to adjust the UI validation. The address is zero or one and a -1 on the power pin indicates no power control.

**Reply by marcobrianza** [reply]
thank you for the clarification
